docs/product-installation-guide/src/main/asciidoc/install-dc-standalone-proc.adoc

```diff
@@ -1,21 +1,43 @@
 [id='install-dc-standalone-proc']
 = Installing standalone {CENTRAL} 
 
-You can use the {CENTRAL} standalone JAR file to install {CENTRAL} in a WildFly Swarm environment. This installation method does not require a separate server installation.
+You can use the {CENTRAL} standalone JAR file to install {CENTRAL} on an application server other than {EAP}. This installation method does not require a separate server installation.
```

Comment:
Talking about installation is a bit misleading in the case of standalone jar. The point of standalone jar is that installation is not need it. You just *run* it.

How about
You can use {CENTRAL} standalone jar file to run {CENTRAL} without needing to deploy it to application server like {EAP}.